### PR TITLE
Tests for Screens

### PR DESCRIPTION
**Pull request by apbuteau**
Added snapshot tests for SplashScreen and BugReportScreen.  In the course of creating the tests, I refactored Button and Text references to use the Core components instead of the @99tx/first-born.

### CODE REVIEW

File: packages/@fbcmobile-ui/package.json [hunk 1]
```diff
@@ -9,7 +9,6 @@
   },
   "license": "BSD-2-Clause",
   "dependencies": {
-    "@99xt/first-born": "^1.1.8",
```
Comment (dalmendray) [inline packages/@fbcmobile-ui/package.json:12]: YESSSS

File: packages/@fbcmobile-ui/Screens/BugReportScreen.js [hunk 2]
```diff
@@ -21,13 +21,12 @@ import React, {useEffect, useRef, useState} from 'react';
...
 import TextFormInput from '@fbcmobile/ui/Components/FormInput/TextFormInput';
 import UserActionLogger from '@fbcmobile/ui/Logging/UserActionLogger';
-import fbt from 'fbt';
-import nullthrows from 'nullthrows';
 import {Colors} from '@fbcmobile/ui/Theme';
 import {EVENT} from '@fbcmobile/ui/Logging/UserActionEvents';
 import {Icon} from 'react-native-material-ui';
 import {Image, ScrollView, View} from 'react-native';
 import {TextInput} from 'react-native';
+import {fbt} from 'fbt';
```
Comment (dalmendray) [inline packages/@fbcmobile-ui/Screens/BugReportScreen.js:29]: I don't think we need to import this in brackets, we've always done:

`import fbt from 'fbt';`

File: packages/@fbcmobile-ui/Components/FormInput/TextFormInput.js [hunk 3]
```diff
@@ -13,10 +13,10 @@
 
 import InvalidTextView from '@fbcmobile/ui/Components/FormInput/InvalidTextView';
 import React, {useEffect, useState} from 'react';
+import Text from '@fbcmobile/ui/Components/Core/Text';
 import fbt from 'fbt';
```
Comment (dalmendray) [inline packages/@fbcmobile-ui/Components/FormInput/TextFormInput.js:17]: See comment below about using `import {fbt} ...`

File: packages/@fbcmobile-ui/Screens/__tests__/BugReportScreen_test.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+/**
```
Comment (dalmendray) [inline packages/@fbcmobile-ui/Screens/__tests__/BugReportScreen_test.js:1]: ❤️

**Review by dalmendray — APPROVED** [review]
I love this Diff! You rock!

**Review by dalmendray — APPROVED** [review]
Please, remember to update the package.json version + CHANGELOG so that the new version publishes to NPM

**Comment by apbuteau** [comment]
I'm not 100% sure why, but the tests complained about not being able to resolve 'fbt' without using the brackets.

**Comment by apbuteau** [comment]
I'll bump the version in a future diff.

**Pull request merged**